http/oauth2/oauth2.go

```diff
@@ -64,14 +63,18 @@ func (t *token) GetValue() string {
 // with more details and returns nil and false if any error occurs during the introspection
 func introspectRequest(r *http.Request, w http.ResponseWriter, tokenIntro TokenIntrospecter) (context.Context, bool) {
 	// Setup tracing
-	span, ctx := opentracing.StartSpanFromContext(r.Context(), "Oauth2")
```

Comment:
What happened to the "OAuth2"?

Reply:
I changed the span name (or description) to `introspectRequest` as that's what the function does.